manifests/kadalu-operator.yaml

```diff
@@ -149,7 +141,10 @@ rules:
     verbs: ["create", "list", "watch", "delete"]
   - apiGroups: ["csi.storage.k8s.io"]
     resources: ["csidrivers"]
-    verbs: ["create", "delete"]
+    verbs: ["get", "create", "delete"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
Move these changes to `operator.yaml.j2` and use `make gen-manifest` to generate this file